src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/targets/Microsoft.Android.Sdk.ILLink.targets

```diff
@@ -13,49 +13,49 @@ This file contains the .NET 5-specific targets to customize ILLink
       Condition=" '$(PublishTrimmed)' == 'true' "
       AfterTargets="ComputeResolvedFilesToPublishList"
       DependsOnTargets="GetReferenceAssemblyPaths;_CreatePropertiesCache">
+    <PropertyGroup>
+      <_TrimmerDumpDependencies Condition=" '$(LinkerDumpDependencies)' == 'true' ">true</_TrimmerDumpDependencies>
+      <_CustomStepDll>$(MSBuildThisFileDirectory)..\tools\Microsoft.Android.Sdk.ILLink.dll</_CustomStepDll>
```

Comment:
This is a tiny thing and I know its a "non public" property, but could this have a more "Android" name? `_CustomStepDll` seems pretty generic and it could possibly clash with user declared properties... maybe. 

Perhaps `_AndoridLinkerCustomStepAssembly` might be better?

Reply:
Thanks, fixed!